kbase-extension/static/kbase/js/common/error.js

```diff
@@ -97,14 +96,14 @@ define(['common/ui', 'common/html'], (UI, html) => {
             title: 'Error',
             body: div(
                 {
-                    class: 'error-dialog__body',
+                    class: 'kb-error-dialog__body',
                 },
                 [
                     ui.buildPanel({
                         title: title,
                         type: 'danger',
                         body: ui.buildErrorTabs({
-                            preamble: p(preamble),
+                            preamble: preamble,
```

Comment:
edited this slightly so that the preamble now gets automatically wrapped in a `p`, since all instance of this preamble text were like that.